superset-frontend/src/dashboard/actions/hydrate.js

```diff
@@ -56,6 +55,8 @@ import newComponentFactory from 'src/dashboard/util/newComponentFactory';
 import { TIME_RANGE } from 'src/visualizations/FilterBox/FilterBox';
 import { FeatureFlag, isFeatureEnabled } from '../../featureFlags';
 import extractUrlParams from '../util/extractUrlParams';
+import { getUrlParam } from '../../utils/urlUtils';
+import { URL_PARAMS } from '../../constants';
```

Comment:
nit: 
```javascript
import { getUrlParam } from 'src/utils/urlUtils';
import { URL_PARAMS } from 'src/constants';
```

Reply:
This is one of those common cases that we might want to put in our wiki